sonar-scala-plugin/build.gradle

```diff
@@ -47,28 +47,32 @@ jacocoTestReport {
 }
 
 dependencies {
-    compileOnly 'org.sonarsource.api.plugin:sonar-plugin-api'
-    testImplementation 'org.sonarsource.sonarqube:sonar-plugin-api-impl'
-    testImplementation 'org.sonarsource.api.plugin:sonar-plugin-api-test-fixtures'
-    implementation 'org.sonarsource.analyzer-commons:sonar-analyzer-commons'
-    implementation 'org.sonarsource.analyzer-commons:sonar-xml-parsing'
-    implementation 'org.sonarsource.slang:slang-api'
-    implementation 'org.sonarsource.slang:slang-checks'
-    implementation 'org.sonarsource.slang:slang-plugin'
-    implementation('org.scalameta:scalameta_2.13:4.8.10') {
+    compileOnly libs.sonar.plugin.api
+
+    implementation libs.sonar.analyzer.commons
+    implementation  (libs.slang.plugin)
```

Comment:
Is there a reason why this dependency is between parentheses?

Reply:
I will reveft it. It was a leftover of the temporary fix to exclude the assertj dependency